src/style.css

```diff
@@ -0,0 +1,37 @@
+* {
+  box-sizing: border-box;
+  margin: 0;
+  padding: 0;
+}
+
+.container {
+  display: flex;
+  flex-direction: column;
+  width: 96;
+  height: 96px;
```

Comment:
And you don't need this rules, your container already has this size because of its children